### ISSUE TITLE
CORS redirects

### ISSUE THREAD

**Issue by iamnathanwindsor** [recall-photos/recall-app#3]
Hi Guys-

Great project!

Any idea how to fix this?

Failed to load resource: the server responded with a status of 404 ()
logger.js:52 [DEBUG] getFile photos.json returned 404, returning null
gaia.blockstack.org/hub/191aftN8eVEefMpWsYnYeU6nUK9mh65iP9/photos.json:1 Failed to load resource: the server responded with a status of 404 ()
logger.js:52 [DEBUG] getFile photos.json returned 404, returning null
logger.js:52 [DEBUG] uploadToGaiaHub: uploading images/elfo.letter-paper.jpg to https://hub.blockstack.org
dashboard:1 Access to fetch at 'https://hub.blockstack.org/store/191aftN8eVEefMpWsYnYeU6nUK9mh65iP9/images/elfo.letter-paper.jpg' from origin 'https://app.recall.photos' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
dashboard:1 Uncaught (in promise) TypeError: Failed to fetch
logger.js:52 [DEBUG] uploadToGaiaHub: uploading compressed_images/elfo.letter-paper.jpg to https://hub.blockstack.org
dashboard:1 Access to fetch at 'https://hub.blockstack.org/store/191aftN8eVEefMpWsYnYeU6nUK9mh65iP9/compressed_images/elfo.letter-paper.jpg' from origin 'https://app.recall.photos' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.

**Comment by alvesjtiago** [comment]
Thank you @iamnathanwindsor! 😊

Are you getting this error on https://app.recall.photos/ or when running locally?
Hope I can help you track down the issue.

**Comment by jackveiga** [comment]
Closing the issue since some time has passed and the discussion has stopped. 

Feel free to open a new issue if the problem persists.
